src/main/java/me/profelements/dynatech/items/misc/DimensionalHomeDimension.java

```diff
@@ -32,24 +35,4 @@ public ChunkData generateChunkData(World world, Random random, int chunkx, int c
         return chunkData;
     } 
 
-    @Override
-    public boolean shouldGenerateCaves() {
-        return false;
-    }
-
-    @Override
-    public boolean shouldGenerateDecorations() {
-        return false;
-    }
-
-    @Override
-    public boolean shouldGenerateStructures() {
-        return false;
-    }
-
-    @Override
-    public boolean shouldGenerateMobs() {
-        return false;
-    }
-
```

Comment:
Why were these removed? None of that should happen when generating the DimensonalHomeDimension

Reply:
those are the default values

Reply:
Ohhhhh I didnt know that.